x-pack/legacy/plugins/maps/public/layers/styles/vector/vector_style.js

```diff
@@ -277,6 +295,47 @@ export class VectorStyle extends AbstractStyle {
   }
 
   _getFieldRange = (fieldName) => {
+    const dynamicProps = this.getDynamicPropertiesArray();
+    const dynamicProp = dynamicProps.find(dynamicProp => { return fieldName === dynamicProp.getField().getName(); });
+
+    if (dynamicProp && dynamicProp.isFieldMetaEnabled()) {
+      let dataRequestId;
+      if (dynamicProp.getFieldOrigin() === FIELD_ORIGIN.SOURCE) {
+        dataRequestId = SOURCE_META_ID_ORIGIN;
+      } else {
+        const join = this._layer.getValidJoins().find(join => {
+          const matchingField = join.getRightJoinSource().getMetricFieldForName(fieldName);
+          return !!matchingField;
+        });
+        if (join) {
+          dataRequestId = join.getSourceMetaDataRequestId();
+        }
+      }
+
+      if (dataRequestId) {
+        const styleMetaDataRequest = this._layer._findDataRequestForSource(dataRequestId);
+        if (styleMetaDataRequest && styleMetaDataRequest.hasData()) {
+          const data = styleMetaDataRequest.getData();
+          const field = dynamicProp.getField();
+          const realFieldName = field.getESDocFieldName ? field.getESDocFieldName() : field.getName();
+          const stats = data[realFieldName];
+          if (stats) {
+            const sigma = _.get(dynamicProp.getFieldMetaOptions(), 'sigma', 3);
+            const stdLowerBounds = stats.avg - (stats.std_deviation * sigma);
+            const stdUpperBounds = stats.avg + (stats.std_deviation * sigma);
+            const min = Math.max(stats.min, stdLowerBounds);
+            const max = Math.min(stats.max, stdUpperBounds);
+            return {
```

Comment:
this return is buried 4-blocks deep.

for readability, either use a guards that return early with the default or use `if-else statement` to align returns.